src/bsoncxx/document/element.cpp

```diff
@@ -78,32 +78,28 @@ stdx::string_view element::key() const {
     return stdx::string_view{key};
 }
 
-// CXX-1817; deprecation warning suppressed for get_utf8()
-BSONCXX_SUPPRESS_DEPRECATION_WARNINGS_BEGIN
 #define BSONCXX_ENUM(name, val)                                     \
     types::b_##name element::get_##name() const {                   \
         types::bson_value::view v{_raw, _length, _offset, _keylen}; \
         return v.get_##name();                                      \
     }
 #include <bsoncxx/enums/type.hpp>
 #undef BSONCXX_ENUM
-BSONCXX_SUPPRESS_DEPRECATION_WARNINGS_END
 
-types::b_utf8 element::get_string() const {
+types::b_utf8 element::get_utf8() const {
     types::bson_value::view v{_raw, _length, _offset, _keylen};
-    return v.get_string();
+    BSONCXX_SUPPRESS_DEPRECATION_WARNINGS_BEGIN
+    return v.get_utf8();
```

Comment:
Can this call the new `v.get_string()` instead ?

Reply:
done